src/lib/accessibility/interfaces.ts

```diff
@@ -108,11 +108,28 @@ export interface IAccessibilityAttributes extends IAriaWidgetAttributes {
 }
 
 export type AccessibilityAttributes = { [partName: string]: IAccessibilityAttributes }
-export type AccessibilityKeyHandlers = { [partName: string]: AccessibilityKeyHandlers }
-
+export type ActionsDefinition = { [partName: string]: IActionDefinition }
 export interface IAccessibilityDefinition {
   attributes?: AccessibilityAttributes
-  keyHandlers?: AccessibilityKeyHandlers
+  actionsDefinition?: ActionsDefinition
+}
+
+export interface IActionDefinition {
+  keyCombinations: KeyCombinations[]
+}
+
+export interface KeyCombinations {
+  keyCode: number
+  shiftKey?: boolean
```

Comment:
I'd rather use default values instead of optional, as this will cause compiler error when we enable strict null check options.

Reply:
@smykhailov Make sense. But as it's interface, we'll need to set these values for each key combination. Or you meant some other usage?

Reply:
I think it is OK to leave it as it is. This should work with strict null checks, see the condition in `keyboardHandlerFilter`